<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>110</y>
      <width>471</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="NumEvent">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code de l'evennement:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date: </string>
      </property>
     </widget>
     <widget class="QLabel" name="type">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type d'evennement:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nbinit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre d'invités:</string>
      </property>
     </widget>
     <widget class="QLabel" name="heure">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>heure de début:</string>
      </property>
     </widget>
     <widget class="QLabel" name="duree">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>durée:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lieu">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>lieu:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leNumEvent">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>61</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="letype">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>111</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lenbinvit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leduree">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>311</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lelieu">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>361</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leheure">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mdf">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="supprimer">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter le code de l'evennements à supprimer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>code QR</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 179, 179);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Gestion des Evennements</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>1031</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page_2">
     <attribute name="label">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tabEvenement">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>1021</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1031</width>
       <height>401</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Recherche par numero</string>
     </attribute>
     <widget class="QLineEdit" name="rechercher">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>inserez le numero de l'evennement:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prechercher">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>101</y>
        <width>831</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <attribute name="label">
      <string>Recherche par lieu</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>inserez le lieu:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherchelieu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewlieu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>101</y>
        <width>831</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <attribute name="label">
      <string>Recherche par type</string>
     </attribute>
     <widget class="QTableView" name="tableViewtype">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>731</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>inserez le type:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherchertype">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>11</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1031</width>
       <height>401</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Tri Evenement</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choisir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NumEvent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lieu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nbinvit</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>trier selon:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>tri croissant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>tri decroissant</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabletri">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>831</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1507</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
